packages/scripts/CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## Unreleased
 
-## Breaking Changes
+### Breaking Changes
```

Comment:
This is somewhat clever that Prettier caught this

Reply:
No, this was me when I was adding a Prettier upgrade notice to the `@wordpress/scripts` changelog 🙂